__tests__/app.test.js

```diff
@@ -0,0 +1,35 @@
+const db = require("../db/connection.js")
+const testData = require("../db/data/test-data")
+const seed = require("../db/seeds/seed.js")
+
+const app = require("../src/app.js")
+
+const request = require("supertest")
+
+beforeEach(() => seed(testData))
+afterAll(() => db.end())
+
+describe("/api/topics", () => {
+    describe("GET", () => {
+        test("200: responds with a body containing all topics", () => {
+            return request(app)
+                .get("/api/topics")
+                .expect(200)
+                .then(({ body: { topics } }) => {
+                    expect(topics.length).toBe(3)
+                    expect(topics).toStrictEqual(testData.topicData)
```

Comment:
lets test that each topic each has the keys that we know are going to be in there as this makes in easy to understand what we are testing for.
topics also shouldn't strictly equal the test data as testData won't have topic_id